### ISSUE TITLE
[Bug]: Domain squatting false positives

### ISSUE THREAD

**Issue by ak47uk** [CyberDrain/Check#145]
### Required confirmations before submitting

- [x] **I can reproduce this issue on the latest released version** of Check.
- [x] **I have searched existing issues** (both open and closed) to avoid duplicates.
- [x] I am **not** requesting general support; this is an actual bug report.

### Issue Description

Since yesterday, several legit sites are triggering "Security threat detected: Domain squatting detected: Domain adds prefix/suffix to protected domain", the protected domain is always "unifi.ui.com". There is a banner at the top of the page:

<img width="459" height="85" alt="Image" src="https://github.com/user-attachments/assets/3e81b2d6-49d6-45f1-b2c7-43a2f924c3f4" />

Example sites that trigger the warning:
plugin.intuitcdn.net
entrabackup.dropsuite.uk
qbo.intuit.com
qbo-preload.app.intuit.com
qfp.intuit.com
accounts.intuit.com

### Extension Version

1.2.0

### Rules Version

1.2.0

### Relevant Logs / Stack Trace

```plaintext
{
  "action": "warn",
  "confidence": 0.7,
  "detected": true,
  "protectedDomain": "unifi.ui.com",
  "severity": "medium",
  "techniques": [
    {
      "confidence": 0.7,
      "description": "Domain adds prefix/suffix to protected domain",
      "pattern": "generic_combo",
      "prefix": "drops",
      "suffix": "te",
      "technique": "combosquat"
    }
  ],
  "testDomain": "entrabackup.dropsuite.uk"
}



{
  "browserInfo": {
    "browserType": "edge",
    "browserVersion": "147.0.0.0",
    "extensionId": "knepjpocdagponkonnbggpcnhnaikajg",
    "installType": "admin",
    "language": "en-GB",
    "platform": "Win32",
    "version": "1.2.0"
  },
  "isManaged": true,
  "profileId": "removed",
  "timestamp": "2026-04-15T07:13:46.657Z",
  "userInfo": {
    "accountType": "work-school",
    "email": "removed",
    "emailNotAvailable": false,
    "id": "removed",
    "provider": "unknown",
    "reason": null
  }
}
```

**Comment by Zacgoose** [comment]
Please mark squatting as disabled in the mean time

**Comment by ak47uk** [comment]
It looks like it is just ultra sensitive and is treating `*.*ui*.*` as squatting of `unifi.ui.com`.

**Comment by homotechsual** [comment]
Also seeing this:

Page URL: https://sanctuarystartswithus.good.do/act/write-to-your-Senedd/?embedded=
Hostname: sanctuarystartswithus.good.do
Timestamp (UTC): 2026-04-15T08:28:39.287Z
Banner Title: ⚠️ Suspicious Domain Detected
Displayed Reason: This website's domain looks similar to "zoom.us" but is NOT the legitimate site. Be careful entering any credentials.
Detection Score: N/A

Detected Indicators:
Domain squatting techniques detected:
levenshtein: Domain differs by 2 character(s) from protected domain

**Comment by Zacgoose** [comment]
Make sure you update your rules as those extra domains have been removed.

**Comment by erikvdberg86** [comment]
We can reproduce this on 1.2.0 / rules 1.2.0 with a legitimate Citrix/NetScaler login page:

URL REDACTED

Blocked as:
Domain Squatting: This site closely resembles "office.com"

Additional notes:

Reproduced in both Chrome and Edge
Extension is managed
URL allowlist is present in registry/policy
Tested with wildcard and exact URL allowlist entries, but the site is still blocked

**Comment by ak47uk** [comment]
Any update on this? Even when I access router/printer admin configs using their local IP, every embedded frame has a banner.

**Comment by homotechsual** [comment]
Also still seeing this consistently and "disable domain squatting" as a solution really is a very Microsoft-y ConnectWise-y resolution - security tech bugged out and completely not useful? Turn it off of course!

**Comment by homotechsual** [comment]
For clarity we're managing this via CIPP

**Comment by Zacgoose** [comment]
fixed in dev
